src/condor_utils/submit_utils.cpp

```diff
@@ -2202,7 +2202,7 @@ int SubmitHash::SetPeriodicExpressions()
 	if ( ! pec)
 	{
 		/* user didn't have one, so add one */
-		if ( ! job->Lookup(ATTR_PERIODIC_HOLD_CHECK)) {
+		if ( ! job->Lookup(ATTR_PERIODIC_HOLD_CHECK) && param_boolean("SUBMIT_INSERT_DEFAULT_POLICY_EXPRS", false)) {
```

Comment:
don't like to see param calls in leaf functions.  this should moved into the submitHash struct.